thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/merge/AnomalyMergeConfig.java

```diff
@@ -0,0 +1,31 @@
+package com.linkedin.thirdeye.anomaly.merge;
+
+public class AnomalyMergeConfig {
+  AnomalyMergeStrategy mergeStrategy = AnomalyMergeStrategy.FUNCTION;
```

Comment:
sequentialAllowedGap is the gap which if exceeded, the anomalies don't need to be merged?
And is mergeDuration the max duration for which anomalies can be merged?
Can we add javadoc/comments to explain these?